### PR TITLE
Launchpad refresh

### PR DESCRIPTION
**Pull request by makevoid**
upgrade launchpad dependencies

TODO after this PR
- restore sentry tracking
- restore user routes (not urgent)

### CODE REVIEW

**Review by diogopalhais — COMMENTED** [review]
Looks Ok. Just left a couple comments on things I think need to be fixed.

File: package.json [hunk 1]
```diff
@@ -7,34 +7,34 @@
...
-  "author": "fabio@appliedblockchain.com",
-  "packageManager": "yarn@3.2.0",
+  "author": "fabio@appliedblockchain.com, francesco@appliedblockchain.com (makevoid@gmail.com)",
+  "packageManager": "yarn@3.6.3",
   "scripts": {
     "start": "GIT_HASH_VERSION=$(git rev-parse HEAD) docker-compose -f docker-compose.local.yml up",
     "start:build": "GIT_HASH_VERSION=$(git rev-parse HEAD) docker-compose -f docker-compose.local.yml up --build",
     "start:prod": "GIT_HASH_VERSION=$(git rev-parse HEAD) docker-compose -f docker-compose.yml up --build",
-    "start:web": "yarn workspace @launchpad-ts/web start & yarn workspace @launchpad-ts/ui-library build:watch",
+    "start:web": "yarn workspace @launchpad-ts/web start & yarn workspace build:watch",
```
[the remark this answers is not in the document]
Comment (makevoid) [reply package.json]: fixed 90d0e36

File: package.json [hunk 2]
```diff
@@ -7,34 +7,34 @@
...
     "start:api": "yarn workspace @launchpad-ts/api start:dev",
-    "storybook": "yarn workspace @launchpad-ts/ui-library storybook",
+    "storybook": "yarn workspace storybook",
```
[the remark this answers is not in the document]
Comment (makevoid) [reply package.json]: removed 194e748

File: services/api/src/controller/auth.ts [hunk 3]
```diff
@@ -56,14 +57,15 @@ export default class UserController {
...
     const [user]: User[] = await userRepository.find({
-      where: { email: body.email },
+      where: { email: email },
     })
     const valid =
-      user && (await passwordHelper.validatePassword(user, body.password))
+      user && (await passwordHelper.validatePassword(user, password))
     if (valid) {
-      const { password, ...userWithoutPassword } = user
+      const { ...userWithoutPassword } = user
```
[the remark this answers is not in the document]
Comment (makevoid) [reply services/api/src/controller/auth.ts]: ah, right, reverted a855c08

**Pull request merged**